docs/sources/http_api/dashboard_versions.md

```diff
@@ -65,6 +67,64 @@ Status Codes:
 - **401** - Unauthorized
 - **404** - Dashboard version not found
 
+## Get all dashboard versions by dashboard UID
+
+Query parameters:
+
+- **limit** - Maximum number of results to return
+- **start** - Version to start from when returning queries
+
+`GET /api/dashboards/uid/:uid/versions`
+
+Gets all existing dashboard versions for the dashboard with the given `uid`.
+
+**Example request for getting all dashboard versions**:
+
+```http
+GET /api/dashboards/uid/QA7wKklGz/versions?limit=2?start=0 HTTP/1.1
+Accept: application/json
+Content-Type: application/json
+Authorization: Bearer eyJrIjoiT0tTcG1pUlY2RnVKZTFVaDFsNFZXdE9ZWmNrMkZYbk
+```
+
+**Example Response**
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json; charset=UTF-8
+Content-Length: 428
+
+[
+  {
+    "id": 2,
+    "dashboardId": 1,
```

Comment:
it is a bit weird that in the response we still have dashboardId, replace by dashboardUid?